docs/learning/WriteAConnector.md

```diff
@@ -2,63 +2,63 @@
 
 ## Table of Contents
 
-- [Table of Content](#table-of-content)
-- [Introduction](#introduction)
-  - [Preface](#preface)
-    - [What is a Connector](#what-is-a-connector)
-  - [Who should read this guide?](#who-should-read-this-guide)
-  - [Ways to sync data to an iTwin](#ways-to-sync-data-to-an-itwin)
-  - [Preliminary Reading](#preliminary-reading)
-  - [Structure of the guide](#structure-of-the-guide)
-  - [Foundations](#foundations)
-    - [iTwin](#itwin)
-    - [iModelHub](#imodelhub)
-    - [iModel](#imodel)
-    - [Briefcases](#briefcases)
-    - [Element](#element)
-    - [Changeset](#changeset)
-- [The basics of writing a Connector](#the-basics-of-writing-a-connector)
-  - [Connecting data to an iTwin](#connecting-data-to-an-itwin)
-    - [Data Extraction](#data-extraction)
-    - [Data alignment](#data-alignment)
-    - [Schemas](#schemas)
-    - [Domain Schemas](#domain-schemas)
-    - [Extending a Schema](#extending-a-schema)
-    - [Dynamic Schemas](#dynamic-schemas)
-    - [Display Labels](#display-labels)
-    - [CodeValues](#codevalues)
-  - [Sync](#sync)
-    - [Detecting and pushing changes](#detecting-and-pushing-changes)
-    - [Provenance and External Repository](#provenance-and-external-repository)
-      - [Case 1 : File metadata](#case-1--file-metadata)
-      - [Case 2 : Id mapping](#case-2--id-mapping)
-    - [Change detection](#change-detection)
-- [Connector SDK](#connector-sdk)
-  - [Getting started](#getting-started)
-  - [BridgeRunner](#bridgerunner)
-  - [Synchronizer](#synchronizer)
-  - [Connector interface methods](#connector-interface-methods)
-    - [InitializeJob](#initializejob)
-    - [OpenSourceData](#opensourcedata)
-    - [ImportDefinitions](#importdefinitions)
-    - [ImportDomainSchema](#importdomainschema)
-    - [ImportDynamicSchema](#importdynamicschema)
-    - [UpdateExistingData](#updateexistingdata)
-  - [Execution Sequence](#execution-sequence)
-  - [Analyzing the Connector output](#analyzing-the-connector-output)
-    - [ECSQL](#ecsql)
-    - [Visualizing the output](#visualizing-the-output)
-  - [Logs](#logs)
-  - [Error Messages](#error-messages)
-  - [Building a test for a Connector](#building-a-test-for-a-connector)
-- [Advanced Topics](#advanced-topics)
-  - [Job Subjects](#job-subjects)
-  - [Schema merging](#schema-merging)
-  - [Units and Coordinate systems](#units-and-coordinate-systems)
-  - [Dealing with geometry](#dealing-with-geometry)
-  - [Authentication](#authentication)
-  - [Locks & Codes](#locks--codes)
-  - [More information](#more-information)
+- [Write A Connector](#write-a-connector)

```

Comment:
Where did this top level come from?

Reply:
It was merged into my working copy of the file when I cherry-picked the commit, as were the package renames and changing `imodeljs` -> `itwinjs-core` in the SDK URL.

Want me to revert all but the `cpx` -> `cpx2` change or keep both that and the URL change?